modules/nw-udn-cr.adoc

```diff
@@ -16,9 +16,9 @@ The following procedure sets up a user-defined network that is namespace scoped.
 
 .Procedure
 
-. Create a request for a user-defined network:
+. Create a request for either a `Layer2` or `Layer3` topology type user-defined network:
 
-.. Create a YAML file, such as `my-udn-request.yaml`, to define your request with content as in the following example:
+.. Create a YAML file, such as `my-layer-two-udn.yaml`, to define your request as in the following example:
```

Comment:
Maybe explicitly call out that `my-layer-two-udn.yaml` is a "layer 2" example. I know the name of the YAML implies that, but...